scripts/ingests/utils.py

```diff
@@ -204,3 +205,35 @@ def ingest_pm(db, sources, muRA, muRA_err, muDEC, muDEC_err, pm_reference, verbo
         n_added += 1
 
     print("Added to database: ", n_added)
+#def ingest_photometry adapted from ingest_pm and ingest_parallaxes from utils 
+def ingest_photometry(db, sources = None, bands = None, ucds = None, magnitudes = None, magnitude_errors = None , telescope = None, instrument = None, epoch = None, comments = None, reference = None, verbose=False):
+    verboseprint = print if verbose else lambda *a, **k: None
+
+    n_added = 0
+
+    for i, source in enumerate(sources):
+        db_name = db.search_object(source, output_table='Sources')[0]['source']
+         #TODO: Check for duplicate photometry 
+        #source_photometry_data = db.query(db.Photometry).filter(db.Photometry.c.source == db_name).table()
+
+        # TODO: Make function which validates refs
+
+        # Construct data to be added
+        photometry_data = [{'source': db_name,
+                          'band': bands[i],
+                          'ucd' : ucds[i],
+                          'magnitude' : magnitudes[i],
+                          'magnitude_error' : magnitude_errors[i],
+                          'telescope': telescope[i],
+                          'instrument': instrument[i],
+                          'epoch': epoch,
+                          'comments': comments,
```

Comment:
This works when `epoch` and `comments` are None but will not have the expected behavior if passed as an array. Try adding the indexes.

Reply:
So I originally had the indexes in there, but it kept tracebacking because it says that Nonetype object wasn't subscriptable, so I took them out for now